packages/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.test.ts

```diff
@@ -1054,38 +1054,47 @@ describe('autocomplete', () => {
       if (fn.name !== 'auto_bucket') {
         for (const signature of fn.signatures) {
           signature.params.forEach((param, i) => {
-            if (i < signature.params.length - 1) {
```

Comment:
These tests were skipping the final parameter of each call signature. Updated to test all parameters of each function.